deploy/distrib/src/main/resources/tomcat/conf/logging.properties

```diff
@@ -23,7 +23,9 @@ handlers = java.util.logging.ConsoleHandler, 1catalina.org.apache.juli.FileHandl
 .handlers = 1catalina.org.apache.juli.FileHandler, java.util.logging.ConsoleHandler
 
 # Allow logging to trace the actual logger producer class:
-java.util.logging.SimpleFormatter.format=%1$tF %1$tT.%1$tL %1$tz %3$s %2$s %n%4$s: %5$s%6$s%n
+java.util.logging.SimpleFormatter.format=%1$tF %1$tT.%1$tL %1$tz %4$s: %3$s %5$s%6$s%n
```

Comment:
an example of before / after could be more relevant to explain the benefit of this change